context/app/static/js/shared-styles/alerts/index.js

```diff
@@ -23,6 +23,7 @@ const StyledAlert = styled(OutlinedAlert)`
   :not(svg) {
     color: ${(props) => props.theme.palette.text.primary};
   }
+  margin-bottom: 16px;
```

Comment:
I don't really like default margins unless it's based on a prop.

Reply:
ok, let me do that. Thanks for the feedback.